sim/core/apl_actions_misc.go

```diff
@@ -137,6 +137,44 @@ func (action *APLActionActivateAuraWithStacks) String() string {
 	return fmt.Sprintf("Activate Aura(%s) Stacks(%d)", action.aura.ActionID, action.numStacks)
 }
 
+type APLActionAddComboPoints struct {
+	defaultAPLActionImpl
+	character *Character
+	numPoints int32
+}
+
+func (rot *APLRotation) newActionAddComboPoints(config *proto.APLActionAddComboPoints) APLActionImpl {
+	character := rot.unit.Env.Raid.GetPlayerFromUnit(rot.unit).GetCharacter()
+	numPoints, err := strconv.Atoi(config.NumPoints)
+	
+	if err != nil {
+		numPoints = 0
+	}
+	return &APLActionAddComboPoints{
+		character: character,
+		numPoints: int32(numPoints),
+	}
+}
+
+func (action *APLActionAddComboPoints) IsReady(sim *Simulation) bool {
+	return true
+}
+
+func (action *APLActionAddComboPoints) Execute(sim *Simulation) {
+	numPoints := strconv.Itoa(int(action.numPoints))
+
+	if sim.Log != nil {
+		action.character.Log(sim, "Adding combo points (%s points)", numPoints)
+	}
+	metrics := action.character.NewComboPointMetrics(ActionID{SpellID: 432264})
```

Comment:
@sanguinerarogue and I talked about this before but this spell ID doesn't really make sense here both for Rogue (who may not even have the Rune equipped) and also since this will be useful for Feral. The problem is that you have to provide an ActionID, but you can't use a blank ID or `SpellID: 0` without breaking things apparently.

Reply:
@sanguinerarogue What you should do is create a new dummy id like we do for Attack/Move/Wait/Etc. It is added in common.proto in `enum OtherAction` and then add it to the switch case in `action_id.ts`